css/css-shapes/spec-examples/shape-outside-010.html

```diff
@@ -32,6 +32,7 @@
     <script src="/resources/testharnessreport.js"></script>
     <script src="support/spec-example-utils.js"></script>
     <script>
+    setup({ single_test: true });
```

Comment:
`approxShapeTest` is also sync, but there's the `onload="checkFloats();"` bit below which makes the suggestion for `verifyTextPoints` unworkable. Instead, can you just pull the `done()` call into the test, after the `approxShapeTest` call?

Reply:
Sure can